### DISCUSSION TITLE
Migrating from Bash to Go on October 14th 2023!

### DISCUSSION THREAD

**Discussion by Link-** [Link-/gh-token#26] [Announcements]
Rewriting this tool in Go has been in the backlog for 1 year and 1 month now: #20 it's time to move to something more reliable and universal than bash.

Thanks to @lindluni we have a working rewrite that's ready to published: #24 

### This announcement is a notification that we'll be migrating to the Go version on **October 14th 2023.** 

Once we do we will be publishing the tool binaries supporting all platforms. All the documentation has been updated and we made sure to maintain parameter backward compatibility.

If you have comments or concerns, this is a great place to leave them.

Thank you for your support and for using this small utility.

**Comment by SunsetYe66** [comment]
@Link- It's November now, any new progress?

**Reply by Link-** [reply]
There are no blockers, I just got swamped. I'll revisit this during this month.

**Reply by Link-** [reply]
It's released now
